### DISCUSSION TITLE
Tessellating in 2D and 3D

### DISCUSSION THREAD

**Discussion by janzenchoi** [neperfepx/neper#532] [Q&A] [answered]
Hello,

Suppose that I were to generate a 3D tessellation and took a very thin slice from it.

Now suppose that I generated a 2D tessellation with similar inputs (e.g., for morphology).

Would the thin slice be statistically the same as the 2D tessellation. In other words, if I used an external program (e.g., MTEX) to extract the statistics from the thin slice and 2D tessellation, would I get a similar distribution for the equivalent diametres, sphericities, etc.? 

Thanks.

**Comment by rquey** [comment, marked as answer]
No, they would not be the same.

Here is for example the case of the equivalent diameters of the cells of a 10000 Voronoi tessellation - 3D and 2D slice:
```
neper -T -n 10000 -statcell diameq -o 3d
neper -T -n 10000 -transform "slice(0.5,0,0,1)" -statcell diameq -o 2d
```
![image](https://user-images.githubusercontent.com/2659376/186381323-770694cc-e1f5-4494-b1e9-9cd8d4340962.png)
